sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -440,6 +440,7 @@ directive:
   transform: >
     $.get.responses["200"]["x-az-response-name"] = "FlattenedDownloadProperties";
     $.get.responses["200"]["x-az-public"] = false;
+    $.get.responses["200"]["x-az-stream"] = true;
```

Comment:
Nit - it's probably good to put this on the 206 response as well because if the order ever gets shuffled, it'd be good if any success type that gets merged together has the flag set.